src/Donors/ValueObjects/DonorType.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Give\Donors\ValueObjects;
+
+use Give\Framework\Support\ValueObjects\Enum;
+
+/**
+ * @since 2.19.6
+ *
+ * @method static DonorType NEW()
+ * @method static DonorType SUBSCRIBER()
+ * @method static DonorType REPEAT()
+ * @method static DonorType SINGLE()
+ * @method bool isNew()
+ * @method bool isSubscriber()
+ * @method bool isRepeat()
+ * @method bool isSingle()
+ */
+class DonorType extends Enum
```

Comment:
Can you rename this to `DonorListTableType`? Reason being it's specific to list tables but not our Donor Model so I don't want anyone getting confused.